src/Stripe.net/Services/PaymentIntents/PaymentIntentConfirmOptions.cs

```diff
@@ -60,9 +60,27 @@ public class PaymentIntentConfirmOptions : BaseOptions
         [JsonProperty("save_payment_method")]
         public bool? SavePaymentMethod { get; set; }
 
+        /// <summary>
+        /// Indicates that you intend to make future payments with this PaymentIntent’s payment
+        /// method.
```

Comment:
Feel free to ignore, but if you want this comment to be rendered as different paragraphs, you must wrap each paragraph in a `<para></para>` tag.

Reply:
Yeah I think I'll ignore for this one but noted.